zerver/tests/tests.py

```diff
@@ -2129,6 +2130,11 @@ def test_generate_204(self):
         result = self.client_get("/api/v1/generate_204")
         self.assertEqual(result.status_code, 204)
 
+    def test_authors_page(self):
+        # type: () -> None
+        result = self.client_get("/authors/")
+        self.assertIn('Contributors', result.content.decode('utf8'))
```

Comment:
You should use `self.assert_in_success_response` here -- it'll check for 200 status and take care of the UTF issue.